docs/howto/admin/systemd.rst

```diff
@@ -0,0 +1,77 @@
+.. _howto/admin/systemd:
+
+================================
+Customizing ``systemd`` services
+================================
+
+By default, TLJH configures two ``systemd`` services to run JupyterHub and Traefik.
+
+These services come with a default set of settings, which are specified in
+`jupyterhub.service <https://github.com/jupyterhub/the-littlest-jupyterhub/blob/master/tljh/systemd-units/jupyterhub.service>`_ and
+`traefik.service <https://github.com/jupyterhub/the-littlest-jupyterhub/blob/master/tljh/systemd-units/traefik.service>`_.
+They look like the following:
+
+.. code-block:: bash
+
+  [Unit]
+  Requires=traefik.service
+  After=traefik.service
+
+  [Service]
+  User=root
+  Restart=always
+  WorkingDirectory=/opt/tljh/state
+  PrivateTmp=yes
+  PrivateDevices=yes
+  ProtectKernelTunables=yes
+  ProtectKernelModules=yes
+  Environment=TLJH_INSTALL_PREFIX=/opt/tljh
+  ExecStart=/opt/tljh/hub/bin/python3 -m jupyterhub.app -f jupyterhub_config.py --upgrade-db
+
+  [Install]
+  WantedBy=multi-user.target
+
+
+However in some cases, admins might want to have better control on these settings.
+
+For example when mounting shared volumes over the network using Samba, these namespacing settings might be a bit too strict
```

Comment:
Maybe add a link to Samba here, so that people can learn more about it?

Reply:
Sounds good!